packages/website/docs/_samples/main/DateRangePicker/DateFormat/sample.html

```diff
@@ -23,11 +23,22 @@
     >
     </ui5-daterange-picker>
 
+    <ui5-daterange-picker
```

Comment:
the sample is not present in the website portal; 
you should add `.md` file, and update the `DateRangePicker.mdx` file;

you can use this [PR](https://github.com/SAP/ui5-webcomponents/pull/9694/files#diff-2127efdcf72309f96938901d40808a73f2bc76dc8c3e5d3c5871a8e259d06f25) for example

Reply:
They're added to the "Date Formats" sample: 
<img width="914" alt="Screenshot 2024-08-19 at 1 17 47 PM" src="https://github.com/user-attachments/assets/84723bc2-2794-48dc-afb6-81b121c30747">